expo-zustand-styled-components/src/components/FileTree/FileTree.tsx

```diff
@@ -1,54 +1,55 @@
 import { useEffect } from 'react';
-import { TouchableOpacity, View } from 'react-native';
+import { View } from 'react-native';
 
 import { useRepoInfoStore } from '../../hooks/stores';
 import getRepoTree from '../../services/get-repo-tree';
 
 import { Cell, LinkText, Containter } from './FileTree.styles';
 import { colors } from '../../utils/style-variables';
 import { FolderIcon, DocumentIcon } from '../Icons';
+import LinkButton from '../LinkButton/LinkButton';
 
-const FileTree = () => {
-  const { path, owner, name, tree, branch } = useRepoInfoStore();
+const FileTree = (props: { path?: string; branch?: string }) => {
+  const { owner, name, tree, branch } = useRepoInfoStore();
```

Comment:
i belive the owner and name should be gotten from the URL params if we are really moving the stored values to the url

Reply:
It isn't the same as the web normal url params. the owner and name were defined as the base URL, and only the base navigator(RepoNavigator) has access to them.